docs/tutorial.rst

```diff
@@ -459,7 +459,7 @@ Don't forget to add the stylesheet link to the templates:
 Working with JSON
 -----------------
 
-In the last part of the tutorial,
+As our last modification,
```

Comment:
```suggestion
As a finishing touch,
```

Reply:
I'm not comfortable calling it a "touch" because it sounds like a cosmetic change but this was part of the problem definition and was also shown in the screenshot. But I agree that it needs rewording.